### PR TITLE
AP-110 Streamline add friend functionality

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

**Which issue(s) this PR fixes** 🔨

Fixes #

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: components/tailored/friends/add/Add.vue [hunk 1]
```diff
@@ -29,6 +30,14 @@ export default Vue.extend({
     }
   },
   methods: {
+    _searchFriend: _.debounce(async function(this:any) {
+      if(this.accountID.length >= 40) {
+        await this.searchFriend()
+      } else {
+        this.error = this.$t('friends.invalid_id') as string
+        this.searching = false
```
Comment (WanderingHogan) [inline components/tailored/friends/add/Add.vue:38]: can you change this to a guard clause

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
please update to use a guard clause, otherwise good

File: components/tailored/friends/add/Add.vue (hunk 1 above)
Comment (davidchan1219) [reply components/tailored/friends/add/Add.vue]: I changed it.

File: components/tailored/friends/add/Add.vue [hunk 2]
```diff
@@ -29,6 +30,13 @@ export default Vue.extend({
     }
   },
   methods: {
+    _searchFriend: _.debounce(async function(this:any) {
+      if(this.accountID.length >= 40) {
```
Comment (RetroPronghorn) [inline components/tailored/friends/add/Add.vue:34]: `if (` missing space after if

**Review by RetroPronghorn — CHANGES_REQUESTED** [review]

**Review by WanderingHogan — APPROVED** [review]
Looks good, just need matts change request

**Pull request closed without merging**